### PR TITLE
Izi refac Administration Menu (Users / Address book)

### PR DESCRIPTION
**Pull request by muarachmann**

### CODE REVIEW

**Comment by muarachmann** [comment]
I havent yet handled the toast i need to get the correct instances when there is a successful save (update in the database) before i can call the toast appropriately. figuring it out though

**Comment by aethelwulffe** [comment]
Please rebase this to account for the /custom directory being cleaned up.

**Comment by aethelwulffe** [comment]
@muarachmann do you have label permissions on the repo (you should).  If this is still a WIP, please tag it as such.  Something like this is not nice to have sitting around being ignored.  I have the urge to test and merge.

**Comment by aethelwulffe** [comment]
Please fix branch...this big commit.  Should finish these up all together in the order submitted.

**Comment by muarachmann** [comment]
@aethelwulffe i dont yet have label permissions

**Comment by muarachmann** [comment]
hi @aethelwulffe please help me with rebasing i fixed the other but this is giving me headache

**Comment by aethelwulffe** [comment]
OK, I am going to give you label permissions (check for an invitation) after this evening's phone meeting.  Whoever is there with me will go over all the repo permissions.  I want GSOC and regular contributors to have a pretty high level of permissions, with the intent that we will also regularly downgrade folks if they are not recently active (but can be upgraded again later).  Unfortunately, Adding labels is not separated from repo write-level permission at this time within GitHUB.  I hear that is in the works though.

The custom/import_xml.php you will find is an old legacy file that was moved to modules/MIPS/boneyard.  It might have been better if I had just deleted it, but if you revert your changes to that file, or just relocate it in your branch to where it lives now, that should clear that up.

I assume that you have looked at the command line instructions link above.  Did ` git merge --no-ff muarachmann-iziRefacUsers` not work for you?

**Comment by muarachmann** [comment]
ok i will try

**Comment by aethelwulffe** [comment]
Test sat.
-We just need to fix this branch, or create a new branch and manually merge the changes into it.  I could pull it as a branch onto this repo if you like, but for GSOC it needs to be your PR.

File: interface/patient_file/summary/add_edit_issue.php [hunk 1]
```diff
@@ -348,15 +349,27 @@ function ActiveIssueCodeRecycleFn($thispid2, $ISSUE_TYPES2) {
...
   // Close this window and redisplay the updated list of issues.
-  echo "<html><body><script language='JavaScript'>\n";
+  echo "<html><body>\n";
+  require_once('../../globals.php');
+  require_once($GLOBALS['srcdir'].'/headers.inc.php');
+  call_required_libraries(array("jquery-min-3-1-1", "font-awesome" , "iziModalToast"));
+  echo "<script language='JavaScript'>\n";
   if ($info_msg) echo " alert('$info_msg');\n";
-//  echo "alert('successfully added new issue');\n";
+  echo "var alertMsg = 'successfully added new issue';\n";
```
[the remark this answers is not in the document]
Comment (muarachmann) [reply interface/patient_file/summary/add_edit_issue.php]: hahaha okay!

**Comment by muarachmann** [comment]
oohhhhh @aethelwulffe had a hard time fixing this havent slept till now time to take a nap

**Comment by muarachmann** [comment]
@tmccormi , @teryhill @aethelwulffe @apooravc  please review this

**Comment by muarachmann** [comment]
@aethelwulffe I touched users too check that to edit a user and create a user
Also i touched issues from the demographics (patients page) and also popups> issues

**Comment by aethelwulffe** [comment]
Mua,

It might be nice to port the changes to a new branch off of a cleanly rebased repo.
There are 36 commits here.  They can be squashed, and maybe it is not worth doing that at this point, but getting things straight here may help other branches.
$ git checkout muabranchwhatever
$ git rebase master

The above is supposed to work.  Make sure to save your working files elsewhere first.
If it does not, just do a local difference tool to merge over the code changes to a new branch off of a completely up-to-date master head.  This is not the "git" way of doing it, but it always works, as it is like doing all the work all at one time in a single commit, pushing the changes, and doing a PR in just a few minutes time.  As we have looked this over quite a lot, we would be able to get this merged quickly.
Despite the text changes that I feel it needs, I am approving this content.  I prefer that we merge a clean branch, as that attracts a second review more easily...something I need before it is permitted to merge.

**Comment by muarachmann** [comment]
ok

**Comment by muarachmann** [comment]
@aethelwulffe i will rather do a cleaner one and i could handle this with my current gsoc task. Also please see my message on the chat forums

**Comment by aethelwulffe** [comment]
@muarachmann Sorry Mua, but I don't actively participate in the chat, and we are not supposed to be using PM's during GSOC, even when it makes perfect sense.

**Comment by aethelwulffe** [comment]
This is a bit messy, but can we please get a review?

**Comment by aethelwulffe** [comment]
@muarachmann PLEEZE don't merge any more commits into your master branch!  🐛

**Comment by tmccormi** [comment]
The scope of this pull request is way beyond User/Address book.  What is this for?  I'm sorry I'm late to this but there is no description on the pull request (just a subject that is incorrect).   I have not idea what I'm supposed to test or approve or exactly what the purpose is.  The screen shot look nearly identical to the originals, what was improved?  How is using izmodal better the the other modal tools.   I really don;t think changes this deep just to do them is very practical, but maybe I'm missing something in the proposal. Was there an original Issue with discussion about this change?

**Comment by aethelwulffe** [comment]
izimodal replaces fancybox.  This is a long known thing.  It also fixes some serious display issues when you have long lists of users.
  Yes, this PR has a lot of merge commits from previous branches.  Makes the review nearly impossible, but squashing takes care of the commits if we want to merge this.

**Comment by aethelwulffe** [comment]
...And I would really like to get this old, old PR in the bag now.

**Comment by aethelwulffe** [comment]
There is a new PR here, so applying the "Don't Merge" tag here until Mua closes it.

**Comment by aethelwulffe** [comment]
Closing this PR.  izimodal either already done, or needs new.

**Pull request closed without merging**